### PR TITLE
Connection Modal: Update text to clarify when tracking begins.

### PR DESCRIPTION
**Pull request by bubel**
Since some tracking begins upon any attempted connection, we need to
clarify the statement provided alongside our connection (“Set Up
Jetpack”) button so that it is apparent to folks connecting to
WordPress.com.

### CODE REVIEW

File: _inc/client/components/connect-button/index.jsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ export class ConnectButton extends React.Component {
 				{ ! this.props.isSiteConnected &&
 					<p className="jp-banner__tos-blurb">
 					{ __(
-						'By connecting your site you agree to our fascinating {{tosLink}}Terms of Service{{/tosLink}} and to {{shareDetailsLink}}share details{{/shareDetailsLink}} with WordPress.com',
+						'By clicking on the button below, you agree to our fascinating {{tosLink}}Terms of Service{{/tosLink}} and to {{shareDetailsLink}}share details{{/shareDetailsLink}} with WordPress.com.',
```
Comment (beaulebens) [inline _inc/client/components/connect-button/index.jsx]: In the interests of brevity, I'd remove "on" and just say "By clicking the button..."

**Comment by bubel** [comment]
Your penchant for brevity accepted, @beaulebens. Updated.

**Comment by eliorivero** [comment]
The text in this dialog should be updated too

captura de pantalla 2018-03-19 a la s 10 43 23

It's in `wp-content/plugins/jetpack/functions.global.php`

**Comment by oskosk** [comment]
@dereksmart in that banner you updated, there are two buttons, not just one. The text then sound be a little bit more vague in this situation `By clicking the button below, you agree to our fascinating...`.

Do you think it's ok like that or there should be a more specific text here ?

**Comment by dereksmart** [comment]
@oskosk good point, I reverted the change to the banner for more consideration.  Let's get this one in as-is, as the title of it specifically intends to only change the modal connection view.  cc @eliorivero

**Comment by oskosk** [comment]
Alright @dereksmart . Moved this one to Milestone 6.0. Will merge when tests finish

**Pull request merged**